templates/perfalert_inactive_regression_needinfo.txt

```diff
@@ -0,0 +1,5 @@
+It has been over {{ extra["nweeks"] * 7 }} days with no activity on this performance regression. :{{ nickname }}, since you are the author of the regressor, bug {{ extra[bugid]["regressor_id"] }}, which triggered this performance alert, could you please provide a progress update?
```

Comment:
nit: usually we start a new line when addressing the user:
```suggestion
It has been over {{ extra["nweeks"] * 7 }} days with no activity on this performance regression.
:{{ nickname }}, since you are the author of the regressor, bug {{ extra[bugid]["regressor_id"] }}, which triggered this performance alert, could you please provide a progress update?
```